### ISSUE TITLE
Only grayscale image is shown

### ISSUE THREAD

**Issue by samsonwyt** [akira108/MinimumOpenCVLiveCamera#2]
It seems that only gray image is displayed in the camera view.

**Comment by samsonwyt** [comment]
I am using Swift 4.

**Comment by akira108** [comment]
Yes. you can rewrite the codes converting camera image into gray with anything you want.

And sorry, this project is old and I didn’t have time to migrate to Swift 4...

2017/10/08 1:36、samsonwyt <notifications@github.com>のメール:

**Comment by samsonwyt** [comment]
I mean I didn't really change to code, with 

output.videoSettings = @{(NSString *)kCVPixelBufferPixelFormatTypeKey : @(kCVPixelFormatType_32BGRA)};

and 

Mat mat = Mat(height, width, CV_8UC4, base);

but it still displays grayscale outputs :(

**Comment by samsonwyt** [comment]
even using the official demo
